FirelandsProject/Firelands-Core
# Updating issue and template files

### Changes proposed

- Update the bug report files, using the .yml format and the information in the pull_request_template file.
- Move the README.md file into the .github folder.

### Issues addressed


- Closes

### SOURCE:


### Tests performed


- Github

### How to Test the Changes:


1. After approving the PR, try to create an issue, it should show you a form, with the fields set inside the .yml.
2. You should also change the format of the data that can be used when making a pull request.

### Known issues and TODO list


---

- Thank you for collaborating with the project.
- For an efficient working methodology.
- It is obvious to make long pull request, unless it is necessary.
- A short pull request would be easy to test and approve.
